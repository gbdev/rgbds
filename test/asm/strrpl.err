warning: strrpl.asm(4): [-Wempty-strrpl]
    STRRPL: Cannot replace an empty string
