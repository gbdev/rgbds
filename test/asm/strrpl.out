	ld [hl+], a
hahohahol
hello
world

[bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb][bbbbbbbb]
